lib/pbench/agent/tool_data_sink.py

```diff
@@ -240,7 +259,12 @@ def launch(self):
             f"{self.benchmark_run_dir}/tm/prometheus.yml:/etc/prometheus/prometheus.yml:Z",
             "prom/prometheus",
         ]
-        self.run = subprocess.Popen(args, stdout=prom_logs, stderr=prom_logs)
+        try:
+            self.run = subprocess.Popen(args, stdout=prom_logs, stderr=prom_logs)
+        except Exception as exc:
+            self.logger.error("Podman run process failed: '%s'", exc)
+            self.abort_launch = True
+            return 0
 
         prom_logs.close()
```

Comment:
Actually, here's another place where a `with` would be handy, because we're skipping the close on any exception.